backend/api/Services/AreaService.cs

```diff
@@ -61,65 +61,42 @@ public async Task<IEnumerable<Area>> ReadAll()
             return await GetAreas().ToListAsync();
         }
 
-        private IQueryable<Area> GetAreas()
-        {
-            return _context.Areas.Include(a => a.SafePositions)
-                .Include(a => a.Deck).Include(d => d.Plant).Include(i => i.Installation).Include(d => d.DefaultLocalizationPose);
-        }
-
         public async Task<Area?> ReadById(string id)
         {
             return await GetAreas()
                 .FirstOrDefaultAsync(a => a.Id.Equals(id));
         }
 
-        public async Task<Area?> ReadByInstallationAndName(Installation? installation, string areaName)
-        {
-            if (installation == null)
-                return null;
-
-            return await _context.Areas.Where(a =>
-                a.Name.ToLower().Equals(areaName.ToLower()) &&
-                a.Installation != null && a.Installation.Id.Equals(installation.Id)
-            ).Include(a => a.SafePositions).Include(a => a.Installation)
-                .Include(a => a.Plant).Include(a => a.Deck).FirstOrDefaultAsync();
-        }
-
         public async Task<Area?> ReadByInstallationAndName(string installationCode, string areaName)
         {
             var installation = await _installationService.ReadByName(installationCode);
             if (installation == null)
+            {
                 return null;
+            }
 
-            return await _context.Areas.Where(a =>
-                a.Installation != null && a.Installation.Id.Equals(installation.Id) &&
-                a.Name.ToLower().Equals(areaName.ToLower())
-            ).Include(a => a.SafePositions).Include(a => a.Installation)
-                .Include(a => a.Plant).Include(a => a.Deck).FirstOrDefaultAsync();
+            return await _context.Areas.Where(a => a.Installation.Id.Equals(installation.Id) && a.Name.ToLower().Equals(areaName.ToLower()))
```

Comment:
Installation is nullable, right?
As we do elsewhere the null check shouldn't be removed